plugins/modules/ldap_search.py

```diff
@@ -0,0 +1 @@
+./net_tools/ldap/ldap_search.py
```

Comment:
Are you sure that this is a symlink? It seems that this is a text file with one line of text in it. (Otherwise ansibullbot won't behave that way.)

Reply:
Strange. Actually it is. It also looks exactly the same as the other ldap symlinks there.

Reply:
Indeed, it is. But then, it looks like right now, tests are also passing.